src/fu-common.c

```diff
@@ -364,6 +364,28 @@ fu_common_add_argv (GPtrArray *argv, const gchar *fmt, ...)
 		g_ptr_array_add (argv, g_strdup (split[i]));
 }
 
+/**
+ * fu_common_get_localstatedir:
+ *
+ * Gets the configured local state directory (which may be dynamic)
+ */
+gchar*
+fu_common_get_localstatedir (void)
+{
+	const gchar *runtime_prefix = NULL;
```

Comment:
needs to be "inside" the #ifdef

Reply:
fixed (looks CI wasn't happy about this too)